aspnetcore/blazor/components/prerender.md

```diff
@@ -103,7 +103,28 @@ No compile time error occurs, but a runtime error occurs during prerendering:
 
 This error occurs because the component must compile and execute on the server during prerendering, but <xref:Microsoft.AspNetCore.Components.WebAssembly.Hosting.IWebAssemblyHostEnvironment> isn't a registered service on the server.
 
-If the app doesn't require the value during prerendering, this problem can be solved by injecting <xref:System.IServiceProvider> to obtain the service instead of the service type itself:
+There are several approaches that you can take to address this scenario:
```

Comment:
I think it helps to list the available approaches up front instead of at the end.